.github/workflows/CD.yaml

```diff
@@ -103,7 +103,6 @@ jobs:
           TRUSTEE_BASE_URL: ${{ secrets.TRUSTEE_BASE_URL }}
           TRUSTEE_ENVIRONMENT_NAME: ${{ github.event.inputs.trusteeEnvironmentName }}
           SSH_PRIVATE_KEY: ${{ secrets.SSH_PRIVATE_KEY }}
-          TRUSTEE_CREDENTIALS: ${{ secrets.TRUSTEE_CREDENTIALS }}
```

Comment:
Im Nachinen: Das ist eigentlich Teil von DALA-297 und sollte (und wird auch) in dem Branch bereits gelöscht